CSharp/Library/QnAMaker/QnAMaker/QnAMakerService/QnAMakerAttribute.cs

```diff
@@ -46,13 +46,18 @@ public class QnAMakerAttribute : Attribute
         /// <summary>
         /// The Subscription Key to access the QnA Knowledge Base.
         /// </summary>
-        public readonly string SubscriptionKey;
+        public readonly string AuthKey;
```

Comment:
This is a breaking change.  Can't you just either leave the name the same or add a new one if they really are different?

Reply:
+1